ui/imports/shared/views/chat/MessageContextMenuView.qml

```diff
@@ -98,6 +98,11 @@ StatusPopupMenu {
     onHeightChanged: { root.y = setYPosition(); }
     onWidthChanged: { root.x = setXPosition(); }
     onClosed: selectedUserPublicKey = ""
+    onOpened: {
+        // Trigger x and y position:
+        x = setXPosition()
```

Comment:
It is possible to replace onHeightChanged/onWidthChanged/onOpened with property binding?